hathor/indexes/memory_address_index.py

```diff
@@ -27,15 +28,15 @@
 logger = get_logger()
 
 
-class MemoryAddressIndex(AddressIndex):
+class MemoryAddressIndex(MemoryTxGroupIndex[str], AddressIndex):
     """ Index of inputs/outputs by address
     """
 
     index: DefaultDict[str, Set[bytes]]
 
     def __init__(self, pubsub: Optional['PubSubManager'] = None) -> None:
+        super().__init__()
         self.pubsub = pubsub
-        self.force_clear()
```

Comment:
Unrelated to this line. We have the `force_clear` method implemented in the `MemoryTxGroupIndex` class. Can we remove from this class?

Reply:
Yes, we can. Just did it. Thanks!